tests/visuals/test_visual_app.py

```diff
@@ -229,17 +229,17 @@ def test_missing_columns_validation(self) -> None:
             )
 
         # Missing `Columns.Model` in reco pd.DataFrame
-        with pytest.raises(KeyError):
-            incorrect_reco = pd.DataFrame(
-                {Columns.User: [1, 2, 3, 4], Columns.Item: [3, 4, 3, 4], Columns.Score: [0.99, 0.9, 0.5, 0.5]}
-            )
-            AppDataStorage.from_raw(
-                reco=incorrect_reco,
-                item_data=ITEM_DATA,
-                interactions=INTERACTIONS,
-                is_u2i=True,
-                selected_requests=SELECTED_REQUESTS_U2I,
-            )
+        incorrect_reco = pd.DataFrame(
```

Comment:
Please move this to a separate test with `test_happy_path_with_missing_model`.
And please add assertion on `expected_grouped_reco` data.

Reply:
Hi, there is my suggestion, how to name this test `test_successful_path_with_missing_model`. Added assertion on `expected_grouped_reco`